File: JAVA/CRESCENTE.java
package curso_programacao;

import java.util.Locale;
import java.util.Scanner;

public class CRESCENTE {
public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);	
		Scanner sc = new Scanner(System.in);
		
		int a, b;
		System.out.println("Digite dois numeros: ");
		a = sc.nextInt();
		b = sc.nextInt();
		
		while(a != b) {
			if (a < b) {
			System.out.println("CRESCENTE!");
			}
			else {
				System.out.println("DECRESCENTE!");
			}
			System.out.println("Digite outros dois numeros: ");
			a = sc.nextInt();
			b = sc.nextInt();
			}
		
		sc.close();
}
}

File: Csharp/CRESCENTE.cs
internal class Program {
    private static void Main(string[] args) {

        int a, b;
        Console.WriteLine("Digite dois numeros: ");
        a = int.Parse(Console.ReadLine());
        b = int.Parse(Console.ReadLine());

        while (a != b) {
            if (a < b) {
                Console.WriteLine("CRESCENTE");
            }
            else {
                Console.WriteLine("DECRESCENTE");
            }
            Console.WriteLine("Digite outros dois numeros: ");
            a = int.Parse(Console.ReadLine());
            b = int.Parse(Console.ReadLine());
        }

        
       


    }
}